README.md

```diff
@@ -71,6 +75,42 @@ Download the latest binary executable for your operating system:
   dnf install tektoncd-cli
   ```
 
+  * [Binary RPM package](github.com/tektoncd/cli/releases/download/v0.6.0/tektoncd-cli_0.6.0_Linux-64bit.rpm)
+
+  On any other RPM based distros you can install the rpm directly :
+
+   ```shell
+    rpm -Uvh github.com/tektoncd/cli/releases/download/v0.5.1/tektoncd-cli_0.6.0_Linux-64bit.rpm
+   ```
+
+#### Linux Debs
+
+  * [Ubuntu PPA](https://launchpad.net/~tektoncd/+archive/ubuntu/cli/+packages)
+
+  If you are running on a latest Ubuntu or Debian you would be able to use our TektonCD CLI PPA :
+
+  ```shell
+  apt update;apt install -y gnupg
+  apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 3EFE0E0A2F2F60AA
+  echo "deb http://ppa.launchpad.net/tektoncd/cli/ubuntu eoan main"|sudo tee /etc/apt/sources.list.d/tektoncd-ubuntu-cli.list
+  apt update &&  apt install -y tektoncd-cli
+  ```
+
+  this may works with older releases but that hasn't been tested.
+
+  * [Binary DEB package](github.com/tektoncd/cli/releases/download/v0.5.1/tektoncd-cli_0.6.0_Linux-64bit.deb)
+
+  On any other Debian or Ubuntu based distro, you can simply install the binary package directly with `dpkg` :
+
+  ```shell
+  curl -LO github.com/tektoncd/cli/releases/download/v0.5.1/tektoncd-cli_0.5.1_Linux-64bit.deb
+  dpkg -i tektoncd-cli_0.5.1_Linux-64bit.deb
+  ```
+
+ * Source install
```

Comment:
This is duplicated :sweat_smile: 
And this should be a markdown title `### Source Install`

Reply:
I was going to say a "wasn't me" but it actually is 46b12a6c1 

![Alt Text](https://media.tenor.com/images/c677112c7c7594c35488408263fb94cf/tenor.gif)